app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -407,5 +411,9 @@ def work_history_breaks
     def safeguarding_issues_details_url
       application_form.has_safeguarding_issues_to_declare? ? provider_interface_application_choice_url(application_choice, anchor: 'criminal-convictions-and-professional-misconduct') : nil
     end
+
+    def cache_key(model)
+      "#{model.cache_key_with_version}-#{ENV['SHA']}"
```

Comment:
Should we do `ENV.fetch` here? would need to fix for non-deployed environments but would avoid cache mixups in the event that SHA happened not to be set